README.md

```diff
@@ -53,12 +53,12 @@ Pronounced *semi*.
 
   * Example:
 
-      `s3mi raid my_raid 7 214`
+      `s3mi raid my_raid 3 668`
```

Comment:
The change here represents a better trade off between IOPS and bandwidth for same cost.